docs/tasks.md

```diff
@@ -726,6 +726,8 @@ spec:
 For more information, see [Using `Workspaces` in `Tasks`](workspaces.md#using-workspaces-in-tasks)
 and the [`Workspaces` in a `TaskRun`](../examples/v1beta1/taskruns/workspace.yaml) example YAML file.
 
+Workspaces can be propagated to embedded task specs, not referenced Tasks. For more information, see [Referenced TaskRuns within Embedded PipelineRuns](pipelineruns.md#referenced-taskruns-within-embedded-pipelineruns). 
```

Comment:
Can you also add docs for propagating workspaces from taskruns -> inline task specs?

Reply:
Oh shoot. I thought those existed. But I only had them for pipelinerun. Thanks! I will update it right away.

Reply:
Done.

Reply:
Could you also update the link here to point to the new docs?

Reply:
Actually, this should be added in the PR propagating workspaces in pipeline runs since it references documentation that does not exist in this PR. I will fix it.